<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicVolumeGainWidget</class>
 <widget class="QWidget" name="MusicVolumeGainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐增强界面</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>615</width>
     <height>409</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>音乐增强界面</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>615</width>
     <height>384</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(255,255,255,150);}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QToolButton" name="addFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="addFilesButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="analysisButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="rmFileButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="rmFilesButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="MusicVolumeGainTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>580</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>320</y>
      <width>520</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
    background-color: #05B8CC;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarAll">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>345</y>
      <width>520</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
    background-color: #05B8CC;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>365</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:1px solid #FFF;background:transparent;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>对象 &quot;标准化/最大化&quot; 音量:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="volumeLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>5</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>105</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>db  (默认为 89db)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>文件进度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>345</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>全部进度</string>
    </property>
   </widget>
   <widget class="QLabel" name="functionBottomLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>115</y>
      <width>580</width>
      <height>195</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255,255,255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>115</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>115</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>音轨音量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>115</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>音轨增益</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>115</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑增益</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>115</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑音量</string>
    </property>
   </widget>
   <zorder>addFileButton</zorder>
   <zorder>addFilesButton</zorder>
   <zorder>analysisButton</zorder>
   <zorder>applyButton</zorder>
   <zorder>rmFileButton</zorder>
   <zorder>rmFilesButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>progressBarAll</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>functionBottomLabel</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicVolumeGainTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">musicvolumegainwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
